### ISSUE TITLE
Project clean up

### ISSUE THREAD

**Issue by KeizerDev** [jariz/Dumpert#64]
With the new API we should to a project clean up and get rid of jsoup and use libraries like [Retrofit](http://square.github.io/retrofit/) instead

**Comment by jariz** [comment]
Gonna rename your ticket because obviously we'll clean it up once we integrate the new API.

**Comment by KeizerDev** [comment]
Lol, thanks :smile:

**Comment by KeizerDev** [comment]
I wait for @CytoDev to merge all his branches so I/we can start with it.
Ps. I'm pretty sure we still need jsoup for the login part.

**Comment by jariz** [comment]
@KeizerDev Good point. Afraid we indeed won't be able to ditch jsoup completely, but wherever possible retrofit should be used.

**Comment by CytoDev** [comment]
@KeizerDev Login.java does not use any third party libs. I like to minimise libs in code I write for a good reason. As for merging everything... Yeah need some help on two of those branches.  
  
@jariz jsoup is __only__ used in API.java, and it is easily swappable with standard httprequests. You won't even be needing libraries for most of the methods.
